src/rpc/blockchain.cpp

```diff
@@ -604,6 +604,20 @@ static UniValue SoftForkDesc(const std::string &name, int version, CBlockIndex*
     return rv;
 }
 
+static UniValue BIP9SoftForkDesc(const std::string& name, const Consensus::Params& consensusParams, Consensus::DeploymentPos id)
```

Comment:
Shouldn't this loop throught all the deployments defined in Consensus::Params and be called from somewhere?
Style: why is not indented?